inc/Enqueue.php

```diff
@@ -133,7 +133,7 @@ public function __construct( $appName, $outputPath, $version, $type = 'plugin',
 	 */
 	public function printPublicPath() {
 		$publicPath = $this->getUrl( '' );
-		$jsCode = 'window.__wpackIo' . $this->appName . $this->outputPath . '=\'' . esc_js( $publicPath ) . '\';';
+		$jsCode = 'window.__wpackIo' . sanitize_key( $this->appName . $this->outputPath ) . '=\'' . esc_js( $publicPath ) . '\';';
```

Comment:
Here, instead of using `sanitize_key`, I would rather have a method defined in the class which would be responsible for sanitization. Reason is, the WordPress API is passed through a filter, which plugins may change and would create incompatibility between what the PHP library has to offer and what the JS library expects. Something like

```php
private function sanitize_path( $path ) {
  $path = strtolower( $path );
  return preg_replace( '/[^a-z0-9_\-]/', '', $path );
}
```

Then you do the same operation on the JS library and we have 100% guarantee that both of them are in sync.

Reply:
Good point. Thanks, and it's fixed.